pkg/geo/geomfn/shared_paths.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geomfn
+
+import (
```

Comment:
mind moving this to `topology_operations.go` as well?

Reply:
sharedpaths is[ not a topological operation](http://www.gitta.info/SpatialQueries/en/html/TopoBasedOps_learningObject2.html#:~:text=Topological%20operators%20are%20components%20of,as%20ArcGIS%2C%20Geomedia%20or%20MapInfo.&text=Further%20topological%20operators%2C%20which%20are,corresponding%20data%20structure%2C%20are%20developed.) right ?

Reply:
it's how it's grouped in GEOS, heh!

https://github.com/libgeos/geos/blob/master/capi/geos_c.h.in#L544